opennms-doc/guide-development/src/asciidoc/text/rest/measurements.adoc

```diff
@@ -16,8 +16,9 @@ The following table shows all supported query string parameters and their defaul
 [options="header"]
 |===
 | name               | default   | comment
-| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be (end + start).
-| end                | 0         | Timestamp in milliseconds. If <= 0, the effective value will be the current timestamp.
+| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be `end - start`
```

Comment:
What I understand is:

positive value => start is relative to EPOCH
negative value => start is relative to "end"

the example under the table describes such a behaviour.

Cyr

2016-07-07 9:19 GMT+02:00 mvrueden <notifications@github.com>:

> In opennms-doc/guide-development/src/asciidoc/text/rest/measurements.adoc
> <https://github.com/OpenNMS/opennms/pull/900#discussion_r69860768>:
>
> > @@ -16,8 +16,9 @@ The following table shows all supported query string parameters and their defaul
> >  [options="header"]
> >  |===
> >  | name               | default   | comment
> > -| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be (end + start).
> > -| end                | 0         | Timestamp in milliseconds. If <= 0, the effective value will be the current timestamp.
> > +| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be `end - start`
>
> end + start is definitely correct. The reason for this is, that start can
> be negative. If it were - a negative value would result in end - -start =
> end + start. This seems wrong for the default value of -14400000.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenNMS/opennms/pull/900/files/054de1c38a45a26e4aefc3800756175a1edfc4ed#r69860768>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHzNV7CkQMHGn1yrG32U2TipL2sv15rpks5qTKiJgaJpZM4JGyit>
> .
>